doc/snippets/Microsoft.Data.SqlClient/SqlColumnEncryptionEnclaveProvider.xml

```diff
@@ -21,21 +21,29 @@ the enclave attestation protocol as well as the logic for creating and caching e
             <param name="clientDiffieHellmanKey">A Diffie-Hellman algorithm object that encapsulates a client-side key pair.</param>
             <param name="attestationUrl">The endpoint of an attestation service for attesting the enclave.</param>
             <param name="servername">The name of the SQL Server instance containing the enclave.</param>
+            <param name="customData">The set of extra data needed for attestating the enclave.</param>
+            <param name="customDataLength">The length of the extra data needed for attestating the enclave.</param>
             <param name="sqlEnclaveSession">The requested enclave session or <see langword="null" /> if the provider doesn't implement session caching.</param>
             <param name="counter">A counter that the enclave provider is expected to increment each time SqlClient retrieves the session from the cache. The purpose of this field is to prevent replay attacks.</param>
             <summary>When overridden in a derived class, performs enclave attestation, generates a symmetric key for the session, creates a an enclave session and stores the session information in the cache.</summary>
             <remarks>To be added.</remarks>
         </CreateEnclaveSession>
         <GetAttestationParameters>
+            <param name="attestUrl">The endpoint of an attestation service for attesting the enclave.</param>
```

Comment:
>attestUrl [](start = 25, length = 9)

rename it as attestationUrl. so it's consistent

Reply:
Updated.